students/Jaeyou-Jeong/westagram/users/views.py

```diff
@@ -0,0 +1,33 @@
+import json
+import bcrypt
+
+from django.http import JsonResponse
+from django.views import View
+
+from users.models import User
+from users.checks import email_validation
+
+class Sign(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        PASSWORD_LENGTH = 8
+        try:
+            if len(data['password']) < PASSWORD_LENGTH:
+                return JsonResponse({"message": "INVALID_PASSWORD"}, status=400)
+            if not  email_validation(data['email']):
+                return JsonResponse({"message": "INVALID_EMAIL"}, status=400)
```

Comment:
```suggestion
            if len(data['password']) < PASSWORD_LENGTH:
                return JsonResponse({"message": "INVALID_PASSWORD"}, status=400)
                
            if not  email_validation(data['email']):
                return JsonResponse({"message": "INVALID_EMAIL"}, status=400)
```
if 문 사이에 공백 추가하여 가독성 높여주세요.

Reply:
네 수정했습니다.